README.md

```diff
@@ -1,79 +1,285 @@
+
+## ASB Admin Client
+
+Azure Service Bus Admin Client is used to manage the Service Bus entities. This client can be used to create, update, delete, and list queues, topics, subscriptions, and rules.
+
+The code snippet given below initializes an admin client with the basic configuration and creates a queue in the Azure Service Bus.
+
+```ballerina
+    import ballerinax/asb;
+
+    configurable string connectionString = ?;
+
+    public function main() returns error? {
+        asb:Administrator adminClient = check new (connectionString);
+
+        asb:QueueProperties? queue = check adminClient->createQueue("test-queue");
+
+        if queue is asb:QueueProperties {
+            log:printInfo(queue.toString());
+        } else {
+            log:printError(queue.toString());
+        }
+    }
 ```
 
-2. To build the Ballerina package:
-   Change the current directory to the ballerina home directory and execute the following command.
+## ASB Message Sender Client
+Azure Service Bus Message Sender Client is used to send messages to a queue, topic, or subscription.
+
+The code snippet given below initializes a message sender client with the basic configuration and sends a message to the Azure Service Bus.
+
+```ballerina
+    import ballerinax/asb;
+
+    configurable string connectionString = ?;
+
+    // Sender Configurations
+    ASBServiceSenderConfig senderConfig = {
+        connectionString: connectionString,
+        entityType: QUEUE,
+        topicOrQueueName: "myQueue"
+    };
 
-```shell script
-    bal pack
+    public function main() returns error? {
+        asb:MessageSender queueSender = check new (senderConfig);
+
+        string stringContent = "This is My Message Body"; 
+        byte[] byteContent = stringContent.toBytes();
+        int timeToLive = 60; // In seconds
+        asb:ApplicationProperties applicationProperties = {
+            properties: {a: "propertyValue1", b: "propertyValue2"}
+        };
+
+        asb:Message message = {
+            body: byteContent,
+            contentType: asb:TEXT,
+            timeToLive: timeToLive,
+            applicationProperties: applicationProperties
+        };
+
+        check queueSender->send(message);
+
+        check queueSender->close();
+    }
 ```
 
-3. To run the tests after build:
-   Change the current directory to the ballerina home directory and execute the following command.
+## ASB Message Receiver Client
+
+Azure Service Bus Message Receiver Client is used to receive messages from a queue, topic, or subscription.
+
+The code snippet given below initializes a message receiver client with the basic configuration and receives a message from the Azure Service Bus.
+
+```ballerina
+    import ballerina/log;
+    import ballerinax/asb;
+
+    configurable string connectionString = ?;
+
+    // Receiver Configurations
```

Comment:
No need to add comments to the code samples.

Reply:
Changed with [ff89865](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/206/commits/ff89865bcc09cd1e41a02929615f7d6cff1ffdca).